test/verify/check-shell-host-switching

```diff
@@ -319,7 +319,7 @@ class TestHostSwitching(testlib.MachineCase, HostSwitcherHelpers):
         # fast that something goes wrong with serving files,
         # especially when those files are large in the "devel"
         # scenario.
-        self.allow_journal_messages(".*: failure while serving external channel: internal-error")
+        self.allow_journal_messages(".*: external channel failed: internal-error")
```

Comment:
Oops, it seems [I was wrong](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19668-ba487574-20240325-115002-fedora-39-devel/log.html#384) -- The message still does exist:
```
src/ws/cockpitchannelresponse.c:        g_debug ("%s: failure while serving external channel: %s", self->logname, problem);
```
So instead I suggest
```
 self.allow_journal_messages(".*external channel.*: internal-error")
```